mitodl/mit-open
# set csrf cookie name from env var

### What are the relevant tickets?
Fixes an issue with https://github.com/mitodl/mit-open/pull/1419

### Description (What does it do?)
This PR allows us to set the CSRF cookie name based on an environment variable. A followup to #1418 that did the same for sessionid. 

### How can this be tested?
In `shared.local.env`, set `CSRF_COOKIE_NAME=some-value` and restart your containers. 

Make an authenticated POST/PATCH request from frontend (creating a userlist, editing your profile). Check that the CSRF cookie sent in the request has expected name.

### Additional Context
- [ ] Need new env var for heroku
- [ ] and for github action
